modules/nf-core/kallistobustools/count/main.nf

```diff
@@ -36,12 +37,29 @@ process KALLISTOBUSTOOLS_COUNT {
         -i $index \\
         -g $t2g \\
         $cdna \\
-        $introns \\
+        $intron \\
         -x $technology \\
+        --workflow $workflow_mode \\
         $args \\
         -o ${prefix}.count \\
-        ${reads.join( " " )} \\
-        -m ${memory}G
+        -m ${memory}G \\
+        ${reads.join( " " )}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        kallistobustools: \$(echo \$(kb --version 2>&1) | sed 's/^.*kb_python //;s/positional arguments.*\$//')
+    END_VERSIONS
+    """
+
+    stub:
+    def args    = task.ext.args ?: ''
+    def prefix  = task.ext.prefix ?: "${meta.id}"
+    def cdna    = t1c ? "-c1 $t1c" : ''
+    def intron  = t2c ? "-c2 $t2c" : ''
+    def memory  = task.memory.toGiga() - 1
```

Comment:
```suggestion
    def prefix  = task.ext.prefix ?: "${meta.id}"
```
Since you don't use these variables in the stub code, you can get rid of them. Unless you prefer to have it as similar to the script block as possible?

Reply:
sorry, I am not familiar with these conventions. How would this work if all of these variables are required for kb count?

Reply:
Not sure I understand this: all of these variables go into the count call.

Reply:
Look closely: Those variables are required for the `kb count` call in the the `script:` block above, but not here in the `stub:` block. Only `$prefix` is used below. That's why I can tell even without knowing the kallisto tool, that they are not required here.